testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -87,15 +89,17 @@ public void setup()
                 .addContactPoint(new InetSocketAddress(stargateHost, 9043)).build();
 
         String testName = name.getMethodName();
-        testName = testName.substring(0, testName.indexOf("["));
+        if(testName.contains("[")) {
+			testName = testName.substring(0, testName.indexOf("["));
```

Comment:
when running this `CqlTest` directly form Intellij, this was failing because the `testName` does not contain any `"["`.
With this fix, it is passing.